tests/framework/interface.go

```diff
@@ -60,4 +60,6 @@ type Client interface {
 	UserList() (*clientv3.AuthUserListResponse, error)
 	UserDelete(name string) (*clientv3.AuthUserDeleteResponse, error)
 	UserChangePass(user, newPass string) error
+
+	Txn(compares, ifSucess, ifFail []string, o config.TxnOptions) (*clientv3.TxnResponse, error)
```

Comment:
Looks like you decided on using strings to represent `compares`, `ifSuccess`, `ifFail` which follows more command line interface. In all other methods we tried to follow client interface which would take `[]clientv3.Op` as arguments. 

Have you maybe considered using it instead?

Reply:
i tried to use same way as with other methods in this interface. almost all methods in this interface take arguments like that of command line, but return values as per client interface
```
Get(key string, opts config.GetOptions) (*clientv3.GetResponse, error)
Delete(key string, opts config.DeleteOptions) (*clientv3.DeleteResponse, error)
```
i think command line type arguments make it easy for the interface to work with `e2e` tests which uses etcdctl and `integration` which uses api.